lib/utils.c

```diff
@@ -90,6 +95,22 @@ int read_prop(char *dev, char *prop, long *value)
 	return -1;
 }
 
+/* Parse either a fraction '.3' or percent '30%
```

Comment:
Does this function really work if you pass it a fraction like `.3`? This function divides whatever double is parsed by 100, so I think it would not work for that case.

Reply:
Check whether other uses of `parse_percent()` in the codebase could be calling it with a fraction (like `.3`) instead of a percentage (like `30%`). If not, I think we should remove the comment that it can parse a fraction. If so, then we need to change the function to be able to handle that.

Reply:
Yes, it seems that  `parse_percent()` isn't being called with a fraction so I'll go ahead remove that comment.